api/monitor/client.go

```diff
@@ -43,7 +44,20 @@ func (c *Monitor) SearchAuditEvents(search *AuditEventSearch, opts ...filters.Op
 	events := &response.ResultSet[AuditEvent]{}
 	params := url.Values{}
 
-	for _, opt := range opts {
+	// Define default start time if empty (similar on how UI is handling start time).
+	if search != nil && search.StartTime == "" {
+		search = &AuditEventSearch{
+			StartTime: time.Now().UTC().AddDate(0, 0, -8).Format("2006-01-02T15:04:05.000Z"),
+		}
+	}
+
+	// Set default options, which will be overwritten by opts if defined.
+	options := append([]filters.Option{
+		filters.Paging(0, 50),
+		filters.Sort("created", "ASC"),
```

Comment:
in the UI the default params is sortkey -> created, sortDir -> DESC, fuzzycount -> true